nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/test/java/org/apache/nifi/csv/TestCSVRecordReader.java

```diff
@@ -63,9 +63,9 @@ private List<RecordField> getDefaultFields() {
         return fields;
     }
 
-    private CSVRecordReader createReader(final InputStream in, final RecordSchema schema, CSVFormat format) throws IOException {
+    private CSVRecordReader createReader(final InputStream in, final RecordSchema schema, CSVFormat format, final boolean trimDoubleQuote) throws IOException {
```

Comment:
Just like in the case of the constructor of CSVRecordReader, I recommend overloading this method with a variant that does not contain the `trimDoubleQuote` parameter, and it should call this method with `trimDoubleQuote=true`.